### DISCUSSION TITLE
Set Checkbox Margin is danger!

### DISCUSSION THREAD

**Discussion by zy1075984** [AvaloniaUI/Avalonia#14712] [General]
Set Checkbox Margin is danger! like this Margin="0,0,-7,0"
the Programe crash!

**Comment by rabbitism** [comment]
Which version? How to reproduce?

**Comment by zy1075984** [comment]
11.8











无来去处

---Original---
From: "Dong ***@***.***&gt;
Date: Fri, Feb 23, 2024 22:38 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [AvaloniaUI/Avalonia] Set Checkbox Margin is danger! (Discussion#14712)




 
Which version? How to reproduce?

**Reply by timunie** [reply]
Sorry I am not able to reproduce. Please file a minimal sample
